morf-postgresql/src/main/java/org/alfasoftware/morf/jdbc/postgresql/PostgreSQLDialect.java

```diff
@@ -168,21 +169,51 @@ protected String getSqlFrom(Cast cast) {
 
   @Override
   protected Collection<String> internalTableDeploymentStatements(Table table) {
+    return ImmutableList.<String>builder()
+        .addAll(createTableStatement(table, null, false))
+        .addAll(createCommentStatements(table))
+        .build();
+  }

```

Comment:
I am not sure I like the call to `createTableStatement(table, null, false)` from here.
You probably saved some code duplication, but then you need to worry about `asSelect != null`, etc.
It feels hacky, perhaps somewhat unnecessarily?